src/v/iceberg/tests/gen_test_iceberg_manifest.py

```diff
@@ -14,69 +14,83 @@
-                                       value_type=IntegerType()))),
-    NestedField(field_id=11,
-                required=True,
-                name="location",
-                field_type=ListType(element_id=12,
-                                    element_required=True,
-                                    element_type=StructType(
-                                        NestedField(field_id=13,
-                                                    required=False,
-                                                    name="latitude",
-                                                    field_type=FloatType()),
-                                        NestedField(field_id=14,
-                                                    required=False,
-                                                    name="longitude",
-                                                    field_type=FloatType())))),
-    NestedField(field_id=15,
-                required=False,
-                name="person",
-                field_type=StructType(
-                    NestedField(field_id=16,
-                                name="name",
-                                required=False,
-                                field_type=StringType()),
-                    NestedField(field_id=17,
-                                name="age",
-                                required=True,
-                                field_type=IntegerType()),
-                )))
+    NestedField(field_id=1, required=False, name="foo", field_type=StringType()),
+    NestedField(field_id=2, required=True, name="bar", field_type=IntegerType()),
+    NestedField(field_id=3, required=False, name="baz", field_type=BooleanType()),
+    NestedField(
+        field_id=4,
+        required=True,
+        name="qux",
+        field_type=ListType(
+            element_id=5, element_required=True, element_type=StringType()
+        ),
+    ),
+    NestedField(
+        field_id=6,
+        required=True,
+        name="quux",
+        field_type=MapType(
+            key_id=7,
+            key_type=StringType(),
+            value_id=8,
+            value_type=MapType(
+                key_id=9,
+                key_type=StringType(),
+                value_id=10,
+                value_required=True,
+                value_type=IntegerType(),
+            ),
+        ),
+    ),
+    NestedField(
+        field_id=11,
+        required=True,
+        name="location",
+        field_type=ListType(
+            element_id=12,
+            element_required=True,
+            element_type=StructType(
+                NestedField(
+                    field_id=13, required=False, name="latitude", field_type=FloatType()
+                ),
+                NestedField(
+                    field_id=14,
+                    required=False,
+                    name="longitude",
+                    field_type=FloatType(),
+                ),
+            ),
+        ),
+    ),
```

Comment:
🥹

Reply:
Call me pervert but I like that. Having opening and closing brackets on the same indent improves readability, whether it's nested calls and round brackets or nested blocks and curly ones in C-like languages. It makes functional style as readable as imperative.

Reply:
I think the emoji from Oren means "it's beautiful" 😃